src/main/java/org/fundacionjala/coding/Ruber/movies/Customer.java

```diff
@@ -0,0 +1,117 @@
+package org.fundacionjala.coding.Ruber.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class Customer.
+ */
+class Customer {
+    private final String name;
+    private static List<Rental> rentals = new ArrayList<Rental>();
+    private static final int REGULAR_AMOUNT = 2;
+    private static final double CHILDRENS_AMOUNT = 1.5;
+    private static final double NORMAL_FINE = 1.5;
+    private static final double RELEASE_FINE = 3;
+    private static final int REGULAR_DAYS_RENTED = 2;
+    private static final int CHILDRENS_DAYS_RENTED = 3;
+
+    /**
+     * Constructor.
+     * @param name The customer's name
+     */
+    public Customer(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * This method add a new rental to the vector.
+     * @param arg The rental
+     */
+    public void addRental(final Rental arg) {
+        rentals.add(arg);
+    }
+
+    /**
+     * This method returns the customer's name.
+     * @return customer's name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * This method calculates runs the entire vector to calculate.
+     */
+    public void statement() {
+        double totalAmount = 0;
+        int frequentRenterPoints = 0;
+        //Enumeration rentals = this.rentals.toArray();
+        for (Rental each : rentals) {
+            double thisAmount = calculateAmount(each);
+            frequentRenterPoints += getFrequentRenterPoints(each);
+            //result += getResult(result, each, thisAmount);
+            totalAmount += thisAmount;
+        }
+        getResult(rentals, totalAmount, frequentRenterPoints);
+    }
+
+    /**
+     * To print the result.
+     * @param rentals The rentals list
+     * @param totalAmount Total amount for the rentals
+     * @param frequentRenterPoints Total frequent renter points for the rentals
+     */
+    public void getResult(final List<Rental> rentals, final double totalAmount, final int frequentRenterPoints) {
```

Comment:
How you will test this method?

you should return a String
Please use a StringBuilder instead of using System.out.print